.gitignore

```diff
@@ -1,3 +1,6 @@
+# Application folders
+legislation_analysis/data/
```

Comment:
Moved this to the top so that it's easier to find and group application folders.